dev-qt/qtwebengine/qtwebengine-5.11.1.recipe

```diff
@@ -0,0 +1,207 @@
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	qtwebengine${secondaryArchSuffix}_devel = $portVersion compat >= 5
+	devel:libQt5WebEngine$secondaryArchSuffix = $portVersion compat >= 5
+	devel:libQt5WebEngineCore$secondaryArchSuffix = $portVersion compat >= 5
+	devel:libQt5WebEngineWidgets$secondaryArchSuffix = $portVersion compat >= 5
+	"
+
+REQUIRES_devel="
+	haiku$secondaryArchSuffix
+	qtwebengine$secondaryArchSuffix == $portVersion base
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libavcodec$secondaryArchSuffix
+	devel:libgcc$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libdbus_1$secondaryArchSuffix
+	devel:libegl$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libEGL$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libharfbuzz_icu$secondaryArchSuffix
+	devel:libhunspell$secondaryArchSuffix
+	devel:libhyphen$secondaryArchSuffix
+	devel:libicuuc$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:liblcms2$secondaryArchSuffix
+	devel:libjsoncpp$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libnss3$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:liborc_0.4$secondaryArchSuffix
+	devel:libpcre2_16$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libprotobuf$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Multimedia$secondaryArchSuffix
+	devel:libQt5Positioning$secondaryArchSuffix
+	devel:libQt5Sensors$secondaryArchSuffix
+	devel:libQt5WebChannel$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5MultimediaWidgets$secondaryArchSuffix
+	devel:libre2$secondaryArchSuffix
+	devel:libsnappy$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libsrtp$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libvpx$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libyasm$secondaryArchSuffix
+	devel:libxcomposite$secondaryArchSuffix
+	devel:libxcursor$secondaryArchSuffix
+	devel:libxi$secondaryArchSuffix
+	devel:libXrandr$secondaryArchSuffix
+	devel:libxtst$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:bison
+	cmd:clang
+	cmd:find
+	cmd:flex
+	cmd:as$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:gperf
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:ninja
```

Comment:
Why is `cmd:ninja` here?

Reply:
Because you need to run qmake, then make, which will run python to bootstrap gn, which generates ninja files, so you can build and use gn to create more ninja files. Does that make sense? ;)